### PR TITLE
Changed bubble sort to std::sort for efficiency

### PR DESCRIPTION
**Pull request by DevMohamedMI**
O(n^2) to O(nlogn) worst case efficiency

### CODE REVIEW

File: modules/perception/fusion/base/sensor_data_manager.cc [hunk 1]
```diff
@@ -110,13 +110,9 @@ void SensorDataManager::GetLatestFrames(
...
   }
 
-  for (size_t i = 0; i < frames->size() - 1; ++i) {
-    for (size_t j = i + 1; j < frames->size(); ++j) {
-      if ((*frames)[j]->GetTimestamp() < (*frames)[i]->GetTimestamp()) {
-        std::swap((*frames)[j], (*frames)[i]);
-      }
-    }
-  }
+  std::sort((*frames)->begin(), (*frames)->end(), [](const auto& frame1, const auto& frame2 ){
```
Comment (lianglia-apollo) [inline modules/perception/fusion/base/sensor_data_manager.cc]: Please fix format error

Comment (DevMohamedMI) [reply modules/perception/fusion/base/sensor_data_manager.cc]:
```suggestion
  std::sort((*frames).begin(), (*frames).end(), [](const auto& frame1, const auto& frame2 ){
```

Comment (lianglia-apollo) [reply modules/perception/fusion/base/sensor_data_manager.cc]: please also check on lint test. ```bash apollo.sh lint```

Comment (lianglia-apollo) [reply modules/perception/fusion/base/sensor_data_manager.cc]:
```modules/perception/fusion/base/sensor_data_manager.cc:113:  Lines should be <= 80 characters long  [whitespace/line_length] [2]
modules/perception/fusion/base/sensor_data_manager.cc:113:  Add #include <algorithm> for sort  [build/include_what_you_use] [4]```

**Pull request closed without merging**